sensor/common/registry/registry_store_test.go

```diff
@@ -125,6 +129,21 @@ func TestRegistryStore_MultipleSecretsSameRegistry(t *testing.T) {
 	assert.Equal(t, reg.Config().Password, dceB.Password)
 }
 
+func TestRegistryStore_FailUpsertCheckTLS(t *testing.T) {
+	ctx := context.Background()
+	regStore := NewRegistryStore(alwaysFailCheckTLS)
+	dce := config.DockerConfigEntry{Username: "username", Password: "password"}
+	ns := "namespace"
+
+	// upsert that fails TLS check should error out and NOT perform an upsert
+	require.Error(t, regStore.UpsertRegistry(ctx, ns, fakeImgName.GetRegistry(), dce))
+	require.Nil(t, regStore.store[ns])
+
+	// a subsequent upsert should not return an error and also NOT perform an upsert
+	require.NoError(t, regStore.UpsertRegistry(ctx, ns, fakeImgName.GetRegistry(), dce))
+	require.Nil(t, regStore.store[ns])
```

Comment:
I think these should be `assert`

Reply:
https://github.com/stackrox/stackrox/commit/642be4031dd69b7d9eb121b7b0243c43c31b6e71

Doesn't change the desired outcome for these particular test cases, but made the update.

Reply:
Not 100% if this is the intention, but in my eyes:

`require` -> not the thing being tested and our test relies on this not failing
`assert` -> the thing we are testing